docs/debugging.md

```diff
@@ -0,0 +1,55 @@
+# Debugging Integrations
```

Comment:
I think we can drop this file, we don't include it in new integrations any longer, expecting folks can figure it out on the interwebs.